internal/sinkcores/outputcore/override.go

```diff
@@ -91,6 +83,15 @@ func (c *overrideCore) Check(ent zapcore.Entry, ce *zapcore.CheckedEntry) *zapco
 		if o.Level.Enabled(ent.Level) {
 			return c.Core.Check(ent, ce)
 		}
+		return ce
+	}
+
+	if c.level.Enabled(ent.Level) {
+		return c.Core.Check(ent, ce)
+	}
+
+	if !c.Core.Enabled(ent.Level) {
+		return ce
```

Comment:
I still don't understand why you need to move this block. Either don't move this block, or update the docstrings we have for c.level/etc.

Reply:
I have to move it down because it currently just checks if the given log's level (say WARN) is allowed by the base level of the logger (say INFO). If it is, it doesn't check the overrides at all it just allows it, so it's impossible to increase the required severity of logs via an override.

I'm just moving up the check for overrides first so if an override is found, it is used to determine if the log should be used, and if not, we fall back to the default thresholding behavior.

I'm not sure what docstrings you mean? overrideCore.Level has the doc:
```
	// level is the passed in level before Core was lowered to take into account
	// the overrides.
```
which still seems accurate.

I did update the docstring for `SRC_LOG_SCOPE_LEVEL` to reflect that severity can go up or down.

Reply:
aah sorry I should of reread them :) I had a different idea in my head. LGTM

Reply:
No I appreciated that you pushed back on something that felt off!